web/src/pages/home/homePage.js

```diff
@@ -628,7 +597,7 @@ const Homepage = () => {
         <div className="small-caribbean-countries-container">
           {smallCaribbeanCountries.map((country, index) => (
             <div className="country-card small-caribbean-country-card" key={index}>
-              <a href="https://www.domits.com/home/" target="_blank" rel="noopener noreferrer">
+              <a href="https://www.domits.com/home/" rel="noopener noreferrer">
```

Comment:
Dit moet global zijn, niet specifiek naar domits.com
```suggestion
              <a href="https://www.domits.com/home/" rel="noopener noreferrer">
```